Assets/Script/PositioningTheUI.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEngine.XR;
+
+public class PositioningTheUI : MonoBehaviour
+{
+    public XRNode xrNode = XRNode.Head; // XRNode for user position and rotation
+    public float distanceFromUser = 2f; // Distance of the UI from the user
+    public float heightFromGround = 1.5f; // Height above the ground where the UI should appear
+
+
+    void Start()
+    {
+        // Check if XRNode is valid
+        if (XRSettings.isDeviceActive)
+        {
```

Comment:
die gesamte Funktionalität könnte zur Übersichtlichkeit in eine separate Funktion ausgelagert werden, sodass in der if-Abfrage nur diese eine Funktion aufgerufen wird